pdns/slavecommunicator.cc

```diff
@@ -836,8 +842,24 @@ void CommunicatorClass::slaveRefresh(PacketHandler *P)
         continue;
     }
 
-    if(!ssr.d_freshness.count(di.id)) // what does this mean? XXX
+    if(!ssr.d_freshness.count(di.id)) { // If we don't have an answer for the domain
+      uint64_t newCount = 1;
+      auto failedEntry = d_failedSlaveRefresh.find(di.zone);
```

Comment:
Uhm. Don't recall. Perhaps I was thinking that .find returns a whole object while all you need is a const reference.